test/e2e/apps/network_partition.go

```diff
@@ -43,7 +43,6 @@ import (
 )
 
 const (
-	timeout                = 60 * time.Second
```

Comment:
BTW, why remove this const here?

Reply:
because this const has been defined in another file (disruption.go) in this pkg.

Reply:
Got it, thanks!